### PR TITLE
Extending properties for c, c++, and fortran to list newer ARM cross-compilers

### PR DESCRIPTION
**Pull request by dbroemmel**
This should work once https://github.com/mattgodbolt/compiler-explorer-image/pull/129 is accepted and compilers are built. I hope I got all the paths right.

### CODE REVIEW

**Comment by dbroemmel** [comment]
In addition to ARM, gcc for Power is included in `fortran.amazon.properties`.

All the cross-compilers should be available once built with now merged https://github.com/mattgodbolt/compiler-explorer-image/pull/127 and https://github.com/mattgodbolt/compiler-explorer-image/pull/131

**Comment by Trass3r** [comment]
Nice! I was going to open an issue asking for a trunk build of arm gcc.

**Comment by dbroemmel** [comment]
Well, it won't be a trunk version, but at least a recent one.

File: etc/config/c++.amazon.properties [hunk 1]
```diff
@@ -276,6 +276,24 @@ compiler.arm710.semver=7.2.1
 compiler.arm64g630.exe=/opt/compiler-explorer/arm64/gcc-6.3.0/aarch64-unknown-linux-gnueabi/bin/aarch64-unknown-linux-gnueabi-g++
 compiler.arm64g630.name=ARM64 gcc 6.3.0 (linux)
 compiler.arm64g630.semver=6.3.0
+compiler.armg640.exe=/opt/compiler-explorer/arm/gcc-6.4.0/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-g++
+compiler.armg640.name=ARM gcc 6.4
+compiler.armg640.semver=6.4.0
+compiler.armg730.exe=/opt/compiler-explorer/arm/gcc-7.3.0/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-g++
+compiler.armg730.name=ARM gcc 7.3
+compiler.armg640.semver=7.3.0
```
Comment (mattgodbolt) [inline etc/config/c++.amazon.properties]: typo here! will merge and fix

Comment (dbroemmel) [reply etc/config/c++.amazon.properties]: Oh boy, I've introduced more copy'paste errors than just this one line. There are 4 of this kind in the `c.amazon.properties` and 2 or 3 more in `c++.amazon.properties`. Sorry.

Comment (mattgodbolt) [reply etc/config/c++.amazon.properties]: No worries at all! I think I got them all :)

Now live on beta! https://godbolt.org/beta/

Comment (dbroemmel) [reply etc/config/c++.amazon.properties]: Thanks for this!

I'm afraid there is yet another typo. You caught the `group.gccaarch64` one, but I smuggled a `;` in the `group.cross` where a `:` should be. This probably messes up the ARM64 and Power compilers.
Again, sorry for the terrible copy'paste.

Also, when I look at the compilers for C or C++, the latest version is always first, for Fortran always last. What have I done to cause this?

**Comment by mattgodbolt** [comment]
Pushed another version to beta (with the `;` replaced to a `:` - thanks @RabsRincon ) -- will push to prod later tonight unless I hear not to :)

**Pull request merged**